src/du_parameters/initial_bwp.py

```diff
@@ -19,43 +17,8 @@ class CalculateBWPLocationBandwidthError(Exception):
     pass
 
 
-def get_total_prbs_from_scs(subcarrier_spacing: Frequency) -> int:
-    """Get the number of Physical Resource Blocks (PRBs) based on Subcarrier Spacing (SCS).
-
-    Args:
-        subcarrier_spacing (Frequency): Subcarrier Spacing as Frequency object.
-        Valid values: 15, 30, 60, 120.
-
-    Returns:
-        int: Number of PRBs for given Subcarrier spacing value.
-
-    Raises:
-        ValueError: If the Subcarrier spacing value is not supported.
-        TypeError: If the Subcarrier spacing is not a Frequency object.
-    """
-    scs_to_prbs = {
-        Frequency.from_khz(15): 275,
-        Frequency.from_khz(30): 137,
-        Frequency.from_khz(60): 69,
-        Frequency.from_khz(120): 33,
-    }
-    try:
-        return scs_to_prbs[subcarrier_spacing]
-    except KeyError:
-        supported_subcarrier_spacing = ", ".join(str(freq) for freq in scs_to_prbs)
-        logger.error(
-            "Subcarrier spacing value: %s is not supported. Supported values: %s",
-            subcarrier_spacing,
-            supported_subcarrier_spacing,
-        )
-        raise ValueError(
-            f"Subcarrier spacing value {subcarrier_spacing} is not supported."
-            f"Supported values: {supported_subcarrier_spacing}"
-        )
-
-
-def calculate_initial_bwp(carrier_bandwidth: int, subcarrier_spacing: Frequency) -> int:
-    """Calculate the initial BWP location and bandwidth.
+def get_initial_bwp(carrier_bandwidth: int) -> int:
```

Comment:
```suggestion
@lru_cache
def get_initial_bwp(carrier_bandwidth: int) -> int:
```

This might be useful here too, as it is called multiple times with the same parameters.

Reply:
OK, used.